util/topgen.py

```diff
@@ -278,19 +278,9 @@ def generate_pinmux(top, out_path):
     topname = top['name']
     pinmux = top['pinmux']
 
-    # MIO Pads
-    n_mio_pads = pinmux['io_counts']['muxed']['pads']
-    if n_mio_pads <= 0:
-        # TODO: add support for no MIO case
-        log.error('Topgen does currently not support generation of a top ' +
-                  'without a pinmux.')
-        return
-
-    if 'pinout' not in top:
-        # TODO: add support for no MIO case
-        log.error('Topgen does currently not support generation of a top ' +
-                  'without a pinout configuration.')
-        return
+    # Generation without pinmux and pinout configuration is not supported.
+    assert ('pinmux' in top)
+    assert ('pinout' in top)
```

Comment:
nit: no need for parentheses (here and below)

Reply:
removed